Formula/commandbox.rb

```diff
@@ -1,8 +1,8 @@
 class Commandbox < Formula
   desc "CFML embedded server, package manager, and app scaffolding tools"
   homepage "https://www.ortussolutions.com/products/commandbox"
-  url "https://downloads.ortussolutions.com/ortussolutions/commandbox/5.3.1/commandbox-bin-5.3.1.zip"
-  sha256 "f0e1f3b989a8664c3f7d0f02362ae8b0f3ab0ecd2b923b3ceeb89a4cf46cb614"
+  url "https://downloads.ortussolutions.com/ortussolutions/commandbox/5.4.0/commandbox-bin-5.4.0.zip"
+  sha256 "0be40d16eee995c7b7c51b9d99be0ba0326647e3f546c71cf0ca94c53c43612d"
   license "LGPL-3.0-or-later"
   revision 1
```

Comment:
`revision` is just a way to force an update even when the version hasn't changed, you can read more at https://docs.brew.sh/Formula-Cookbook#formulae-revisions

in this case, it just needs to be removed (and remember to squash your commits please!)
```suggestion
```

Reply:
Can the user who added it remove it?  cc/  @Moisan I'm just trying to update a version here.  I don't quite understand how the build was allowed to pass when it was added if it's invalid.  I'd rather not have to worry about removing stuff random people have added to my formula every time I just need to update a version.

Reply:
It seems simpler for you to just remove it in this PR, no? That's what CI is asking you to do. (It actually also needs to be removed at the same time as the version gets bumped, or else CI will error with `revision should not decrease`.)

We need to add a `revision` line when a formula needs to be rebuilt, typically due to a change in one of its dependencies. If we don't, the formula gets broken as soon a user does `brew upgrade`.

The `revision` line is then removed at every version bump. If you don't like checking for it being there, use `brew bump-formula-pr`.

Reply:
To be more clear - removing the `revision` is part of the version update process.

We add a `revision` when we want to rebuild the package when the upstream software has not released a new version (like a redo on an existing version).

This counter gets reset on every version bump. So if we somehow needed to rebuild `commandbox` 5.4.0 again (for some defect, or other reason), we would then add `revision 1` again. And repeat for any other time we need to rebuild version 5.4.0 (`revision 2`, `revision 3`, etc.). Then on the next update past 5.4.0, the revision would get set to 0 again (in which case the field is just removed).

Reply:
I appreciate the help guys.  What may not be apparent is that my outsider perspective is a little different than yours.  For starters, I don't use Homebrew and I don't even own a Mac.  I only keep this formula updated as a service to my unfortunate users who have inflicted themselves with that :)  I don't really understand Homebrew, nor do I really particularly care about it.  It's just another check on my release schedule, and quite frankly, I dread having to deal with it for this exact sort of reason.  There's ALWAYS something wrong with the formula.  I mean, like _every single time_.  The Homebrew update regularly is the slowest, most annoying part of my release schedule.  (The [new version already released](https://www.ortussolutions.com/blog/commandbox-540-released) to the rest of the world yesterday.)  The builds never pass and it's a new and interesting reason every time that I didn't cause by I have to figure out.  I really wish doing something as menial as pushing a new version could "just work" without needing to learn Ruby and become a Brew expert in whatever random part of your build process has changed since the last time I released.  My S3 sync, apt repos, yum repos, Windows binaries, and even my Brew tap pin repo are 100% automated.  This pull is the only manual step I have to take when I release.

I clicked the commit button above, but I have no idea how to squash commits as I'm just sending this pull via the Github web UI.